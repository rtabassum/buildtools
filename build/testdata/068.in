# buildifier: table sort 1
manifest = [("ab", "bc", "cd"), ("a", "b", "c"), ("", "", ""),]
