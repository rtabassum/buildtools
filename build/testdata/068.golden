# buildifier: table sort 1
manifest = [
   ("a",  "b",  "c"),
   ("ab", "bc", "cd"),
   ("",   "",    ""),
]
